### PR TITLE
Updated the API references overview section

### PR DESCRIPTION
**Pull request by shraddhavp**
this PR is a link to #890 and this PR updates the API overview section

### CODE REVIEW

File: docusaurus/docs/api/overview.mdx [hunk 1]
```diff
@@ -10,38 +10,21 @@ export const Toc = () => <TOCInline toc={toc} />
...
-This section documents the complete Wechaty API.
+API- Application programming interface is a software interface that offers services to other software services. Wechaty API is a very tiny and easy to import API that allows users to use all the functionalities of Wechaty to its best. The API offers a variety of user classes, typedefs, functions, methods, and many more features. This section is an overview of the Wechaty API.
 
-## Top-level Exports
+## Importing the Wechaty API
 
-Every class described below is a top-level export.
+You can import the Wechaty API as shown below:
 
-### Wechaty Class
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/api/overview.mdx:21]: We are removing this from here, but where are we adding this (I mean aren't we retaining this somewhere else?)

File: docusaurus/docs/api/overview.mdx [hunk 2]
```diff
@@ -10,38 +10,21 @@ export const Toc = () => <TOCInline toc={toc} />
...
-
-- [Class Wechaty](./wechaty)
-
-### User Classes
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/api/overview.mdx:25]: Same comment as above

File: docusaurus/docs/api/overview.mdx (hunk 1 above)
Comment (shraddhavp) [reply docusaurus/docs/api/overview.mdx:21]: yes we are in this https://wechaty.js.org/docs/api/wechaty section

File: docusaurus/docs/api/overview.mdx (hunk 2 above)
Comment (shraddhavp) [reply docusaurus/docs/api/overview.mdx:25]: here https://wechaty.js.org/docs/api/wechaty

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/api/overview.mdx [hunk 3]
```diff
@@ -10,38 +10,21 @@ export const Toc = () => <TOCInline toc={toc} />
...
-
-### ES6/TypeScript
+## ES6/TypeScript
 
 ```js
 import { Wechaty } from 'wechaty'
 ```
 
-### ES5 (CommonJS)
+## Javascript
```
Comment (huan) [inline docusaurus/docs/api/overview.mdx]: I'd like to suggest that we can remove all CommonJS related docs from our site.

See: #1275

**Review by proudofsimin — APPROVED** [review]

**Comment by shraddhavp** [comment]
@huan  removed the common JS part from the docs

**Review by huan — APPROVED** [review]
LGTM

**Review by Soumi7 — APPROVED** [review]
LGTM

**Pull request merged**